### PR TITLE
Setting up the managers basic functions and common settings

### PR DESCRIPTION
**Pull request by Aleman778**

### CODE REVIEW

File: web-server/models/prosumer.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    }
+
+    
+
+    /**
+     * Updates the specified fields from the prosumer/user object in the database.
+     * Note: updated_at is automatically updated in the users  with the current time.
+     * @param {array} fields containing strings of each field to include.
+     */
+    update(fields) {
```
Comment (Blinningjr) [inline web-server/models/prosumer.js]: Why have you added this function? I don't see the need to update the prosumers historical data.

File: web-server/views/index.ejs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+          }) %>
+      <div id="header">
+        <div id="headerContent">
+          <h3>Monitor the electricity grid</h3>
+          <strong>or</strong>
+          <h3>Manage the electricity grid</h3>
+        </div>
+      </div>
+      <div id="content">
+          <h1>Hello</h1><br>
```
Comment (Blinningjr) [inline web-server/views/index.ejs]: Shoud there be 13 hello here?

File: web-server/models/prosumer.js (hunk 1 above)
Comment (Aleman778) [reply web-server/models/prosumer.js]: The only field that needs to updated is the house_filename, but I repurposed the prosumer_data to hold current information about the prosumer not historical data. This is so the simulator also can restart and use those values to setup e.g. battery information. But production and consumption is per step basis so this should not be stored here. Do you need the prosumer_data table? It was not used anywhere in the project.

File: web-server/views/index.ejs (hunk 2 above)
Comment (Aleman778) [reply web-server/views/index.ejs]: No this was just to see the background parallax effect but I didn't know what to put there. I will remove this.

**Pull request merged**